docs/configuration/git-providers/gitlab-cloud.md

```diff
@@ -1 +1,33 @@
 # GitLab Cloud
+
+## GitLab Application
+
+Follow the instructions below to set up the Codacy integration with GitLab Cloud:
+
+1.  Create an application pointing to your local Codacy deployment URL.
+
+    You'll need to add the following 'Redirect URI'. Make sure to update your protocol to use either http or https and your domain name as well. Keep in mind this field is case sensitive.
+
+    `https://codacy.example.com/login/GitLab`
+
+    `https://codacy.example.com/add/addProvider/GitLab`
+
+    `https://codacy.example.com/add/addService/GitLab`
```

Comment:
This is wrong, please check the docs on zendesk, I fixed it there

Reply:
Check for the right ones https://support.codacy.com/hc/en-us/articles/360012368539-Configuring-GitLab-com-with-Codacy-Self-hosted

Reply:
I would also put them all in the same block to copy easier together